src/Pages/NotFoundPage/NotFoundPageStyled.js

```diff
@@ -0,0 +1,5 @@
+const NotFoundPage = () => {
```

Comment:
shouldn't be this content in NotFoundPage.js instead of NotFoundPageStyled?

Reply:
Fixed